pkgs/development/python-modules/django-timezone-field/default.nix

```diff
@@ -14,26 +14,25 @@
 
 buildPythonPackage rec {
   pname = "django-timezone-field";
-  version = "5.1";
+  version = "6.1.0";
   disabled = pythonOlder "3.5";
-  format = "pyproject";
+  pyproject = true;
+  build-system = [ poetry-core ];
 
   src = fetchFromGitHub {
     owner = "mfogel";
     repo = pname;
     rev = version;
-    hash = "sha256-FAYO8OEE/h4rsbC4Oc57ylWV7TqQ6DOd6/2M+mb/AsM=";
+    hash = "sha256-vRaWy8DpX64pw1LS/sbnOl/QIpKnnjVO+p7sbKJDVEM=";
   };
 
-  nativeBuildInputs = [ poetry-core ];
-
   propagatedBuildInputs = [
     django
     djangorestframework
     pytz
   ];
 
-  pythonImportsCheck = [ "timezone_field" ];
+  # pythonImportsCheck = [ "timezone_field" ];
```

Comment:
Why is this commented?

Reply:
I think that check's controlled by `dontUsePythonImportsCheck` instead. Is the imports check failing on master?

Reply:
No, the import check succeeds on master. I moved the change here for easier review: https://github.com/NixOS/nixpkgs/pull/325608